python/python/lance/dataset.py

```diff
@@ -3725,6 +3726,9 @@ def write_dataset(
         These row ids are stable after compaction operations, but not after updates.
         This makes compaction more efficient, since with stable row ids no
         secondary indices need to be updated to point to new row ids.
+    file_writer_options : optional, dict
+        Extra file write options that make sense to control lance data file. This is
+        used to set page size for file in disk.
```

Comment:
You should probably document somewhere what the keys are.

Reply:
fine. I will add docs and tests after resolved the ci issues